test/e2e/serial/tests/scheduler_cache.go

```diff
@@ -326,7 +326,7 @@ var _ = Describe("[serial][scheduler][cache][tier1] scheduler cache", Label("sch
 				// In this testcase, having running pods is not good enough: we want to ALSO have pods which keep being
 				// pending "forever". We can't really check "forever", so we just check "long enough".
 
-				deviceName := e2efixture.GetDeviceType1Name()
+				deviceName := e2efixture.GetDeviceType3Name()
```

Comment:
in hindsight, we probably need to either use more descriptive names or the abstract these devices a bit further.
Stuff for a new PR anyway

Reply:
Thanks for the review and the comment. I agree, we set up the devices' settings using a separate CI job and tried as much as possible to include the needed combinations. I'll enhance the naming in a later PR.